src/local_scheduler/lib/python/local_scheduler_extension.cc

```diff
@@ -179,6 +179,55 @@ static PyObject *PyLocalSchedulerClient_set_actor_frontier(PyObject *self,
   Py_RETURN_NONE;
 }
 
+static PyObject *PyLocalSchedulerClient_wait(PyObject *self, PyObject *args) {
+  PyObject *py_object_ids;
+  int num_returns;
+  int64_t timeout_ms;
+  int wait_local;
+
+  if (!PyArg_ParseTuple(args, "Oili", &py_object_ids, &num_returns, &timeout_ms,
+                        &wait_local)) {
```

Comment:
I think the correct way to do this is to parse it as `O` and then use `PyObject_IsTrue`